ext/tidy/tidy.c

```diff
@@ -289,12 +289,12 @@ static int _php_tidy_set_tidy_opt(TidyDoc doc, char *optname, zval *value)
 	zend_long lval;
 
 	if (!opt) {
-		php_error_docref(NULL, E_NOTICE, "Unknown Tidy Configuration Option '%s'", optname);
+		php_error_docref(NULL, E_WARNING, "Unknown Tidy configuration option \"%s\"", optname);
```

Comment:
I wonder if this can be bumped to a ValueError in 8.0.

Reply:
Given that very similar errors below are now ValueError, I think we should indeed do that for the sake of consistency.

Reply:
The only notable difference from the rest is that when this warning is emitted, processing continues, and the option is ignored (it comes from an array). That's the main reason I didn't want to take 2 steps ahead and make it an exception.

Reply:
Hm, yeah, that makes sense.